iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -1093,5 +1100,16 @@ PortManager::acquireConditionVariableData(const RuntimeName_t& runtimeName) noex
     return m_portPool->addConditionVariableData(runtimeName);
 }
 
+bool PortManager::isInternal(const capro::ServiceDescription& service) const noexcept
+{
+    const capro::ServiceDescription serviceRegistry{
```

Comment:
Can we define this one in a similar way to the introspection services? introspection_types.hpp is not the right place, maybe in iceoryx_posh_types.hpp?

If not, we could at least make it `static` here.

Reply:
You're right the internal `popo::Publisher` should be consolidated in one place. In my opinion that should either happen with #518 or #1135 and not on this PR.